src/components/Faq/Faq.vue

```diff
@@ -94,24 +122,109 @@ export default {
   name: "FAQ",
   data() {
     return {
-      questionAndAnswers: [],
-      selectedLanguage: "English",
+      categories: {
+        /*
```

Comment:
I kept this for documentation, let me know if I shall remove this.

Reply:
Documentation is always good. Let's keep it. 🙂